src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -660,6 +688,24 @@ public boolean configure(
 		}
 	}
 
+	@Extension
+	public static class BuildResultTokenMacro extends DataBoundTokenMacro {
```

Comment:
It's up to the user to specify it in one of the fields if they like, i.e. they can set the success message to be "Build ${BUILD_NUMBER} on ${NODE_NAME} was ${BUILD_RESULT}". This is particularly useful since a "successful" build may in fact be unstable as tests may have failed, or aborted by user, etc.

I'll add documentation to that effect in the fields' help files and update the PR.